guide/getting-started/software.md

```diff
@@ -111,15 +114,16 @@ Another reason is that standards are still evolving as some of the technology is
    layout = "float-right-desktop"
 %}
 
-Making purchases with bitcoin is great, as long as we have stores to buy things from. Payment processing applications offer easy-to-use online stores and point-of-sale experiences, paired with more complex feature sets for merchants to manage their inventory, accounting, shipping, and integrations with other services.
+Payment processing applications offer easy-to-use online stores and point-of-sale experiences paired with complex feature sets for merchants to manage their inventories, track their accounts, quickly ship products, and integrate with other services.
 
 </div>
 
 ## Exchanges
 
 <div class="center" markdown="1">
 
-Exchanges allow for swapping between currencies and usually fall in three general categories. Consumer-focused exchanges allow for users to make trades directly with the exchange. This is ideal for occasional use.
+Exchanges lets users swap between currencies and typically fall in three general categories. Consumer-focused exchanges let users make trades directly with the exchange. This is ideal for occasional use.
```

Comment:
Consumer-focused exchanges isn't really that clear. This is referring to simple buy and sell exchanges like Swan Bitcoin yeah?

Reply:
Probably.